db/re/skill_db.yml

```
@@ -18480,6 +18480,16 @@ Body:
     SplashArea: 5
     Duration1: 30000
     Status: Killing_Aura
+  - Id: 784
+    Name: ALL_EVENT_20TH_ANNIVERSARY
+    Description: RO 20th Anniversary Firecracker
+    MaxLevel: 1
+    TargetType: Self
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      AllowOnMado: true
+    Cooldown: 3000
```

```suggestion
    Cooldown: 3000
    Requires:
      SpCost: 1
```
according to skillinfolist.lub